src/hooks/requests/useInstitutions.ts

```diff
@@ -106,3 +107,43 @@ export const useUpdateInstitutionDiscounts = () => {
     isLoading,
   }
 }
+
+export const useInstitutionVacationsFetch = () => {
+  const { isLoading, isError, data } = useQuery<InstitutionVacationsDataType>({
+    queryKey: ['institution-vacations'],
+    queryFn: () => apiClient.get(`${endpoints.INSTITUTION_VACATIONS}`),
+  })
+
+  const { data: institutionVacations } = data || {}
+
+  return {
+    institutionVacations,
+    isLoading: isLoading,
```

Comment:
**[non-blocker]**

You can use shorthand here as well. If you have a variable with the same name as the key name you want for an object, then you can just write 

 return {
    institutionVacations,
    isLoading,
    isError,
  }